### DISCUSSION TITLE
File selection / Cut / Copy / Paste problems in File Explorer?

### DISCUSSION THREAD

**Discussion by leighss** [valinet/ExplorerPatcher#3407] [General]
I am having troubles selecting files and then cut / paste or drag drop problems in File Explorer, erratically, wont work then freezing app
Drag and drop will work on individual files but slowly
ExplorerPatcher appears to be working fine in what I use it for (bar at top) - no updates available

Edition	Windows 11 Home
Version	23H2
Installed on	‎04/‎02/‎2023
OS build	22631.3737
Serial number	
Experience	Windows Feature Experience Pack 1000.22700.1009.0

Anyone else having similar issues?

Could just be a Windows problem ......

PS a 'Copy' option under 'About' would be handy when quoting ExplorerPatcher version

**Comment by pyrates999** [comment]
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.  These should be screen shots.  If you can, please change it to english first.

Please include any other software you have installed that modifies the UI of windows.

You can post about wanting a copy button in the EP properties application as another discussion to make it a feature request.
